subjects/java/checkpoints/config-protector/README.md

```diff
@@ -0,0 +1,68 @@
+## Config Protector
+
+### Instructions
+
+Create a class `ConfigProtector` that provides a method to hide sensitive data in a configuration file using `Regex`. The method should replace sensitive values with asterisks. The configuration file will be provided as a string, and the keys for the sensitive data will be given in a list.
+
+### Expected Class
+
+```java
+import java.util.List;
```

Comment:
Shouldn't these imports be included as well?
```java
import java.util.regex.Matcher;
import java.util.regex.Pattern;
```

Reply:
yeah, done.